android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -235,13 +306,20 @@ public void onMeetingStatusChanged(MeetingStatus meetingStatus, int errorCode, i
     }
   }
 
+
   private void registerListener() {
     Log.i(TAG, "registerListener");
     ZoomSDK zoomSDK = ZoomSDK.getInstance();
     MeetingService meetingService = zoomSDK.getMeetingService();
     if(meetingService != null) {
+      Log.i(TAG, "registerListener meetingService");
       meetingService.addListener(this);
     }
+    InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+    if (inMeetingService != null) {
+      Log.i(TAG, "registerListener inMeetingService");
+      inMeetingService.addListener(new RNZoomUsInMeetingServiceListener(reactContext, inMeetingService));
```

Comment:
Great to have meeting events in a separated file.
However I see that only for android so if used (for consistency)
it would be great to have the same pattern on iOS